src/Microsoft.ML.ImageAnalytics/ExtensionsCatalog.cs

```diff
@@ -97,6 +97,53 @@ internal static ImageLoadingEstimator LoadImages(this TransformsCatalog catalog,
             env.CheckValue(columns, nameof(columns));
             return new ImageLoadingEstimator(env, imageFolder, InputOutputColumnPair.ConvertToValueTuples(columns));
         }
+        /// <summary>
+        /// Create a <see cref="ImageLoadingEstimator"/>, which loads the data from the column specified in <paramref name="inputColumnName"/>
+        /// as an image to a new column: <paramref name="outputColumnName"/>.
+        /// </summary>
+        /// <param name="catalog">The transform's catalog.</param>
+        /// <param name="outputColumnName">Name of the column resulting from the transformation of <paramref name="inputColumnName"/>.
+        /// This column's data type will be <see cref="VectorDataViewType"/>.</param>
+        /// <param name="inputColumnName">Name of the column with paths to the images to load.
+        /// This estimator operates over text data.</param>
+        /// <param name="imageFolder">Folder where to look for images.</param>
+        /// <param name="useImageType">Image type flag - If true loads image as a VectorDataView type else loads image as ImageDataViewType. Defaults to ImageDataViewType if not specified or false.</param>
```

Comment:
>useImageType [](start = 25, length = 12)

if true shouldn't it load as ImageType? but you seem to load as VectorDataView type of "bytes"?

Reply:
With the new naming convention it makes sense. I will change that. Thanks.

Reply:
Changed.